src/share/vm/jwarmup/jitWarmUp.cpp

```diff
@@ -1777,10 +1812,8 @@ PreloadMethodHolder* JitWarmUpLogParser::next() {
   mh->set_size(method_size);
 
   // add class init chain relation
-  /*
-  int method_chain_offset = static_cast<int>(first_invoke_init_order) >= class_chain_offset ? first_invoke_init_order
-                            : _holder->chain()->length() -1;
-                            */
+  // Mount the method to its corresponding declaring class entry in the ClassInitChain
+  tty->print_cr("[JitWarmUp] INFO: Mount the method <%s: %s> to the class %s", method_name_char, method_sig_char, class_name_char);
```

Comment:
add check of PrintCompilationWarmUpDetails

Reply:
Done.